products/identity/Products/Identity Dashboard/Identity_Dashboard/Managing Teams and Users.md

```diff
@@ -0,0 +1,110 @@
+# Managing Teams and Users
+
+## Teams
+
+Teams are the organization units responsible for managing client application integrations with Sign-in Service.
+
+### Adding a Team
+
+Follow these steps to create a team in Identity Dashboard:
+
+* Submit request to the OCTO-Identity team to create a new team in Identity Dashboard.
```

Comment:
🗣️ This is intentionally vague since we don't have a defined request submission process. We'll need to update this (and all other "Submit request to the OCTO-Identity team..." references) to reflect whichever request process we implement (Slackbot, etc.)

I could make this more clear by changing it up to say something like "Submit a request to the OCTO-Identity team via Slack channel #vsp-identity" or something along those lines. Opinions?

Reply:
+1 I think we should make this action-oriented so they can use the doc to do the thing. So if we complete this process via Slack for now, we can use your verbiage and link directly to the Slack channel. If we implement a form or other process in the future, we could update this doc (and other docs) with that info and relevant link!